nilearn/glm/tests/test_contrasts.py

```diff
@@ -264,7 +264,7 @@ def test_deprecation_contrast_type_attribute():
         contrast = Contrast(effect, variance, contrast_type="t")
 
     with pytest.deprecated_call(match="0.13.0"):
-        contrast.contrast_type
+        contrast.contrast_type  # noqa: B018
```

Comment:
Are we expecting a deprecation warning here?

Reply:
It gives the warning if I specify version `0.14.0`, not with `0.13.0`. However the test already does not check if warning is raised.

Reply:
OK had to adapt the stacklevel of the warning but not sure why the test did not catch this.

```python
from nilearn.glm import Contrast
import numpy as np
effect = np.ones((1, 3))
variance = effect[0]
contrast = Contrast(effect, variance, contrast_type="t")
contrast.contrast_type
```

```
/home/remi/github/nilearn/nilearn/tmp/dep.py:9: DeprecationWarning: The parameter "contrast_type" will be removed in 0.13.0 release of Nilearn. Please use the parameter "stat_type" instead.
  contrast = Contrast(effect, variance, contrast_type="t")
/home/remi/github/nilearn/nilearn/tmp/dep.py:11: DeprecationWarning: The attribute "contrast_type" will be removed in 0.13.0 release of Nilearn. Please use the attribute "stat_type" instead.
  contrast.contrast_type
```